cns/restserver/util.go

```diff
@@ -600,6 +600,14 @@ func (service *HTTPRestService) getNetworkContainerDetails(networkContainerID st
 	return containerDetails, containerExists
 }
 
+// areNCsPresent returns true if NCs are present in CNS, false if no NCs are present
+func (service *HTTPRestService) areNCsPresent() bool {
+	if len(service.state.ContainerStatus) == 0 && len(service.state.ContainerIDByOrchestratorContext) == 0 {
+		return false
+	}
+	return true
```

Comment:
just a thought, this pattern is fundamentally 
```go
if (bool) { return false }
return true
```
which is really
```go
return !bool
```

Reply:
Ah I didn't think about that